src/main/java/org/jenkins/tools/test/model/PluginRemoting.java

```diff
@@ -167,24 +167,20 @@ public PomData retrievePomData() throws PluginSourcesUnavailableException {
 
             String parentNode = xpath.evaluate("/project/parent", doc);
             if (StringUtils.isNotBlank(parentNode)) {
-                LOGGER.log(Level.INFO, parentNode);
+                LOGGER.log(Level.FINE, "Parent POM: {0}", parentNode);
```

Comment:
Downgrading this to `FINE`: this is essentially trace level debugging information that isn't relevant to the end user.